debug.log

```diff
@@ -1,2 +1,4 @@
 [1119/110851.627:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie mo�e odnale�� okre�lonej �cie�ki. (0x3)
```

Comment:
Remove this file

Reply:
i can't deleted this file i don't know why but i deleted all it is in this file